integrationtests/src/test/java/io/kroxylicious/proxy/KroxyStandaloneIT.java

```diff
@@ -52,7 +52,8 @@ public void shouldProxyWhenRunAsStandaloneProcess(KafkaCluster cluster, Admin ad
                 new NewTopic(TOPIC_1, 1, (short) 1),
                 new NewTopic(TOPIC_2, 1, (short) 1))).all().get();
 
-        try (var tester = kroxyliciousTester(withDefaultFilters(proxy(cluster)), new SubprocessKroxyliciousFactory(tempDir));
+        ConfigurationBuilder builder = proxy(cluster);
```

Comment:
hmm removing the `withDefaultFilters` using IDEA must have extracted these variables, will inline them tomorrow